README.md

```diff
@@ -88,3 +88,32 @@ gauge('test').set_value(42)
 ```
 
 See `examples/pyformance_usecase.py` for a complete code example using Pyformance.
+
```

Comment:
I wonder if the Handler should have an "async" parameter that users can set to true or false and if false, don't put items on the queue but send them in the call.